src/pages/mynotes/MyNotes.jsx

```diff
@@ -0,0 +1,28 @@
+
+import React from 'react'
+import NoteCard from '../../components/card/NoteCard';
+import Navbar from '../../components/navbar/Navbar';
+import Sidebar from '../../components/sidebar/Sidebar';
+import { CgNotes} from "react-icons/cg"
+
+function MyNotes() {
+
+  return<>
+  <Navbar/>
+  {/* d-flex wd-100 height-100 style={{ display:"flex " }} style={{ width:"100%", }} style={{ height:"100%" ,color:"#E6E6E6" }} */}
```

Comment:
remove unnecessary comments before pushing

Reply:
yes  for sure .